src/index.js

```diff
@@ -327,7 +324,7 @@ class AudioPlayer extends React.Component {
     return (
       <div
         id="audio_player"
-        className={classNames('audio_player', { 'top': this.props.placeAtTop })}
+        className={classNames('audio_player')}
```

Comment:
Looks good. Since we no longer have conditionals this can be changed to just `"audio_player"`.